File: 1159.py
x = 999
while x !=0:
    x=int(input())
    soma = 0
    i = 1
    if x != 0:
        while i <= 5:
            
            if x % 2 == 0:
                soma = soma + x
                x = x + 1
                i = i+ 1
            else:
                x = x + 1
            
            
        print(soma)
    elif x ==0:
        break

File: CSharp/1159.cs
using System;

class URI
{
    static void Main()
    {
        int n;
        int soma=0;
        int cont=0;

        while(true)
        {
            n= int.Parse(Console.ReadLine());
            if(n==0)
            {break;}

            while(cont!=5)
            {
                if(n%2==0)
                {
                    cont++;
                    soma+=n;
                }
                n++;
            }
            Console.WriteLine(soma);
            soma=0;
            cont=0;
        }
    }        
}
